lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_get_latest_release.rb

```diff
@@ -0,0 +1,110 @@
+require 'fastlane/action'
+require_relative '../client/firebase_app_distribution_api_client'
+require_relative '../helper/firebase_app_distribution_auth_client'
+require_relative '../helper/firebase_app_distribution_helper'
+
+module Fastlane
+  module Actions
+    class FirebaseAppDistributionGetLatestReleaseAction < Action
```

Comment:
Consider adding a specs for the action

Reply:
Done! Also added some for the new API client method.